### PR TITLE
Prompt Beta Users to Sign In

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Review
This PR gets very, very explicit about encouraging users to sign in. This is particularly aimed at our Beta users, since the lack of signing in has caused serious issues with the data study.

- Added: "Please SIGN IN" popup on Classifier page - only visible to users who didn't sign in.
  - Quirk: the banner also appears when the user signs out.
  - Quirk: the banner appears behind the Tutorial popup.
- Changed: "Please sign in" banner is now only visible to users who didn't sign in.
  - Banner is also slightly larger.
- Changed: Banner (and the associated stylesheet) has been re-labelled as `//BETA_ONLY` in the code.
- Fixed: App.jsx calls for props.initialised but doesn't map the value properly. (Not that it matters in practice, as it turns out.)

screen shot 2017-11-22 at 16 28 14

_The new popup, visible only on the Classifier page and only when the user hasn't signed in._

screen shot 2017-11-22 at 16 28 12

_Quirk: the user sees the "Please Sign In" popup only after the tutorial._

screen shot 2017-11-22 at 16 28 18

_The "Please Sign In" banner, in teal, at the top of the page. It appears on every page, when the user hasn't signed in._

### Dev Notes
I can make the popup appear in front of the tutorial by setting `.popup.beta-popup-sign-in-prompt { z-index: 1000 }`, but do I want to get into a z-index feud with a ZRC component? It's simple to implement (I've tested it) but isn't tussling over whose z-index trumps whose how World War Z started?

### Status
Ready for review, tagging @wgranger and @beckyrother 

If everyone's off for Thanksgiving already, I'll take responsibility to merge & deploy this ASAP so we can course-correct the sign-in issue. 👌

### CODE REVIEW

**Comment by beckyrother** [comment]
Thanks for doing this so quickly Shaun! Couple things: 
- Is there a way to add a sign-in box in that popup to help users sign in quicker?
- Can the tutorial not come up until after the box is dismissed? Modals on modals is not a great experience. If no, can the box come up over the tutorial?

**Comment by shaunanoordin** [comment]
A sign-in prompt on the Popup itself? That's a great idea! 👍 

I can't avoid the modal on modal issue, but I can at least put the sign-in prompt above the tutorial.

Alright, let's go!

**Comment by shaunanoordin** [comment]
## PR Update
The sign in prompt (popup) has been updated with a new Sign In button:

screen shot 2017-11-22 at 17 42 47

Credit for the design, as always, goes to Becky. 👍 

### Status
Ready to go! @wgranger, be sure to share if you have any comments code-wise! 👍

**Comment by wgranger** [comment]
Do you still need to push up the "continue without signing in" option? I don't see it on this branch yet.

File: src/containers/ClassifierContainer.jsx [hunk 1]
```diff
@@ -227,6 +230,31 @@ class ClassifierContainer extends React.Component {
             {this.state.popup}
           </Popup>
         }
+        
+        {console.log('='.repeat(80), '\n', this.props, this.state)}
```
Comment (wgranger) [inline src/containers/ClassifierContainer.jsx]: Should this log still be in here?

Comment (shaunanoordin) [reply src/containers/ClassifierContainer.jsx]: Nyarp! No it shouldn't, removing now - thanks for the spot!

**Comment by shaunanoordin** [comment]
Did I not push the "Continue without Signing In" update?? Crap - it's not in here, hold on, let me see what happened.

**Comment by shaunanoordin** [comment]
@wgranger Thanks for spotting my mistakes - I absolutely forgot to push the update, and `git status` didn't show me that I was ahead by a few commits since, strangely, my local branch wasn't "hooked up" (?) to the remote branch.

**Comment by shaunanoordin** [comment]
👌  OK, gonna merge and deploy this.

**Pull request merged**